tester/src/test/scala/spinal/tester/scalatest/SpinalSimRegIfTester.scala

```diff
@@ -0,0 +1,233 @@
+package spinal.tester.scalatest
```

Comment:
How about rename this file to AxiLite4RegIfTester or something more specific?

Reply:
The plan was test multiple/all bus types. As there isn't a unified driver/monitor interface and those aren't implement for all buses I'm okay renaming it.